File: src/Example3Java/EX11.java
package Example3Java;

import java.util.Scanner;

public class EX11 {
    public static void main(String[] args) {
        double[] a = new double[10];
        for (int i = 0; i < a.length; i++) {
            System.out.print("Enter number " + (i + 1) + " : ");
            a[i] = new Scanner(System.in).nextInt();
        }
    }
}

File: src/Example3Kotlin/EX11.kt
import java.util.*

object EX11 {
    @JvmStatic fun main(args: Array<String>) {
        var a = DoubleArray(10)
        for (i in a.indices) {
            print("Enter number ${i + 1} : ")
            a[i] = Scanner(System.`in`).nextDouble()
        }
    }
}
